### DISCUSSION TITLE
Adjust location update frequency, as needed

### DISCUSSION THREAD

**Discussion by Noobalicious** [Hyundai-Kia-Connect/kia_uvo#606] [Q&A]
I'm trying to set up an automation that uses the car's location as a trigger, but the device_tracker.my_niro_location entity doesn't refresh nearly often enough for this.  Is there a way to make it temporarily refresh much more often?  Or is there a way to use the sensor.my_vehicle_geocoded_location entity instead for the same purpose?
I am trying to avoid using my phone as the trigger.  Is there a way to have the car do this?

**Comment by cdnninja** [comment]
You can call the force update service.   Careful though you  have a max number of calls per day and it also contacts the car so can kill the battery.

**Comment by Noobalicious** [comment]
That sounded interesting until I realized it just defers the same issue.  (That is, how do I automatically trigger the force call update service?)  I guess I'll consider using family phones or a camera or an Arduino in the car sending an MQTT message.
